### PR TITLE
Implement Initial Atomic Load Bytecodes for Virgil Interpreter

### PR DESCRIPTION
**Pull request by khagankhan**
- Implement the following atomic load bytecodes for the Virgil Interpreter:
  - `I32_ATOMIC_LOAD`
  - `I64_ATOMIC_LOAD`
  - `I32_ATOMIC_LOAD8_U`
  - `I32_ATOMIC_LOAD16_U`
  - `I64_ATOMIC_LOAD8_U`
  - `I64_ATOMIC_LOAD16_U`
  - `I64_ATOMIC_LOAD32_U`
- Add initial regression tests to ensure functionality.

### CODE REVIEW

File: test/regress/ext:threads/atomic_loads.wast [hunk 1]
```diff
@@ -0,0 +1,51 @@
+(module
```
Comment (titzer) [inline test/regress/ext:threads/atomic_loads.wast:1]: Can you split up this test into two files, maybe one for `i32` and one for `i64`? I generally try to keep the regression tests small, or at least significantly smaller than the spec tests, in order to make them easier to debug.

**Review by titzer — APPROVED** [review]
lgtm with minor comment

File: test/regress/ext:threads/atomic_loads.wast (hunk 1 above)
Comment (khagankhan) [reply test/regress/ext:threads/atomic_loads.wast]: Done!

**Comment by titzer** [comment]
You'll need to add these tests to `test/regress.failures.x86-64-linux` until they're implemented in the fast tiers.

**Review by titzer — APPROVED** [review]

**Comment by titzer** [comment]
It failed again; PTAL

**Comment by khagankhan** [comment]
> It failed again; PTAL

There was a subtle mistake  regarding `atomic_load_i64.bin.wast` file path. Tested locally and got the 'ok' results:

```
##+test/regress/ext:threads/atomic_load_i32.bin.wast
##-ok: (ignored expected failure: @ 16:2 assert_return expected 0, got trap[INVALID_OPCODE])
##+test/regress/ext:threads/atomic_load_i64.bin.wast
##-ok: (ignored expected failure: @ 19:2 assert_return expected 0uL, got trap[INVALID_OPCODE])
```

**Pull request merged**